README.md

```diff
@@ -70,9 +70,7 @@ Change history can be found in [`HISTORY.md`](https://github.com/mongodb/node-mo
 
 ### Compatibility
 
-The driver currently supports 3.6+ servers.
-
-** 3.6 support is deprecated and support will be removed in a future version **
+The driver currently supports 4.0+ servers.
```

Comment:
I can move this to its own PR.  I noticed that we never updated this readme after removing support for 3.6 in v6.10: https://github.com/mongodb/node-mongodb-native/releases/tag/v6.10.0